api-extractor/src/definitions/test/ApiDocumentation.test.ts

```diff
@@ -32,31 +32,26 @@ const extractor: Extractor = new Extractor({
   compilerOptions: compilerOptions,
   errorHandler: testErrorHandler
 });
-let myDocumentedClass: ApiStructuredType;
-
-/**
- * Dummy class wrapping ApiDocumentation to test its protected methods
- */
-class TestApiDocumentation extends ApiDocumentation {
-  constructor() {
-    super(myDocumentedClass, extractor.docItemLoader, extractor, console.log);
-  }
-}
 
 extractor.loadExternalPackages('./testInputs/external-api-json');
 // Run the analyzer once to be used by unit tests
 extractor.analyze({
   entryPointFile: path.join(inputFolder, 'index.ts')
 });
 
-myDocumentedClass = extractor.package.getSortedMemberItems()
-  .filter(apiItem => apiItem.name === 'MyDocumentedClass')[0] as ApiStructuredType;
+const myDocumentedClass: ApiStructuredType = extractor.package.getSortedMemberItems()
+.filter(apiItem => apiItem.name === 'MyDocumentedClass')[0] as ApiStructuredType;
 
 describe('ApiDocumentation tests', function (): void {
   this.timeout(10000);
 
   describe('ApiDocumentation internal methods', function (): void {
-    const apiDoc: TestApiDocumentation = new TestApiDocumentation();
+    const apiDoc: ApiDocumentation = new ApiDocumentation(
+      myDocumentedClass.getJsDocs(),
```

Comment:
You could also just put a text string here, e.g.

"Some summary\n@remarks and some remarks\n@public"